src/lib/bootloader/grub2.rb

```diff
@@ -184,34 +184,18 @@ def disk_order_summary
     def locations
       locations = []
 
-      partition_location = boot_partition_location
-      locations << partition_location unless partition_location.empty?
-      if stage1.extended_partition?
-        # TRANSLATORS: extended is here for extended partition. Keep translation short.
-        locations << Yast::BootStorage.extended_partition.name + _(" (extended)")
-      end
+      partition_location = Yast::BootStorage.boot_partitions.map(&:name).join(", ")
+      locations << partition_location + _(" (/boot)") if stage1.boot_partition?
```

Comment:
IMHO this string should not be translated

Reply:
are you sure that `(` and `)` is same in all languages? because e.g. in czech `"` is not same and depends if it is opening or closing one.

Reply:
Hum, you have a point, https://www.mediawiki.org/wiki/Localisation#Symbols,_colons,_brackets,_etc._are_parts_of_messages I'll have to look closer.

Reply:
In practice (in yast-translations) we don't have a language that would use non-ASCII parentheses so this is fine.